docs/regression.ipynb

```diff
@@ -0,0 +1,696 @@
+    }
+   ],
+   "source": [
+    "make_theil_sen_percentiles(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 4) Create a Function to Select a Specific Column"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This function creates a transformation to simply select a specific column from a dataset. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 78,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_select_column(j):\n",
+    "    return dp.t.make_user_transformation(\n",
+    "        input_domain=dp.np_array2_domain(T=float, num_columns=2),\n",
+    "        input_metric=dp.symmetric_distance(),\n",
+    "        output_domain=dp.vector_domain(dp.atom_domain(T=float)),\n",
+    "        output_metric=dp.symmetric_distance(),\n",
+    "        function=lambda x: x[:, j],\n",
+    "        stability_map=lambda b_in: b_in)\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 79,
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "Transformation(\n",
+       "    input_domain   = NPArray2Domain(num_columns=2, T=f64),\n",
+       "    output_domain  = VectorDomain(AtomDomain(T=f64)),\n",
+       "    input_metric   = SymmetricDistance(),\n",
+       "    output_metric  = SymmetricDistance()\n",
+       ")"
+      ]
+     },
+     "execution_count": 79,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "make_select_column(1)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### 5) Create a Function to Make Private Quantiles in Bounds"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "In this section, we will chain a transformation and measurement to compute the median through the existing quantile scores methods. First, we define the discrete_scores transformation by calling dp.t.make_quantile_score_candidates which outputs scores how similar each candidate is to the given alpha-quantile on the input dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 80,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Transformation(\n",
+      "    input_domain   = VectorDomain(AtomDomain(T=f64)),\n",
+      "    output_domain  = VectorDomain(AtomDomain(T=usize), size=100),\n",
+      "    input_metric   = SymmetricDistance(),\n",
+      "    output_metric  = LInfDistance(T=usize)\n",
+      ")\n"
+     ]
+    }
+   ],
+   "source": [
+    "discrete_scores = dp.t.make_quantile_score_candidates(dp.vector_domain(dp.atom_domain(T=float)), \n",
+    "                                    dp.symmetric_distance(), \n",
+    "                                    np.array(list(range(100))).astype(float), \n",
+    "                                    0.5)\n",
```

Comment:
With 3 or more arguments, I'd prefer to name them. If something is just in the wrong order, it can be a pain to debug.

Prefer to just indent all args 4 spaces. Could look into tools for code formatting in notebooks, if that's easier. We've been using flake8 on the main codebase.

Reply:
ok, made the changes. let me know if that's what you meant!